src/main/java/no/nav/pto/veilarbportefolje/oppfolgingsbruker/OppfolgingsbrukerRepositoryV3.java

```diff
@@ -148,27 +143,22 @@ public static OppfolgingsbrukerEntity mapTilOppfolgingsbruker(ResultSet rs, int
             return null;
         }
         return new OppfolgingsbrukerEntity(rs.getString(FODSELSNR), rs.getString(FORMIDLINGSGRUPPEKODE),
-                toZonedDateTime(rs.getTimestamp(ISERV_FRA_DATO)), rs.getString(ETTERNAVN), rs.getString(FORNAVN),
+                toZonedDateTime(rs.getTimestamp(ISERV_FRA_DATO)),
                 rs.getString(NAV_KONTOR), rs.getString(KVALIFISERINGSGRUPPEKODE), rs.getString(RETTIGHETSGRUPPEKODE),
-                rs.getString(HOVEDMAALKODE), rs.getString(SIKKERHETSTILTAK_TYPE_KODE), rs.getString(DISKRESJONSKODE),
-                rs.getBoolean(SPERRET_ANSATT), rs.getBoolean(ER_DOED), toZonedDateTime(rs.getTimestamp(ENDRET_DATO)));
+                rs.getString(HOVEDMAALKODE),
+                toZonedDateTime(rs.getTimestamp(ENDRET_DATO)));
     }
 
     public List<String> finnSkjulteBrukere(List<String> fnrListe, BrukerinnsynTilganger brukerInnsynTilganger) {
         var params = new MapSqlParameterSource();
         params.addValue("fnrListe", fnrListe.stream().collect(Collectors.joining(",", "{", "}")));
-        params.addValue("tilgangTilKode6", brukerInnsynTilganger.tilgangTilAdressebeskyttelseStrengtFortrolig());
-        params.addValue("tilgangTilKode7", brukerInnsynTilganger.tilgangTilAdressebeskyttelseFortrolig());
         params.addValue("tilgangTilEgenAnsatt", brukerInnsynTilganger.tilgangTilSkjerming());
 
         return dbNamed.queryForList("""
-                SELECT fodselsnr from oppfolgingsbruker_arena_v2
-                where fodselsnr = ANY (:fnrListe::varchar[])
-                AND (
-                    (diskresjonskode = '6' AND NOT :tilgangTilKode6::boolean)
```

Comment:
Jeg tror at vi må fremdeles ha en sjekk for diskresjonkode, vi kan hente de fra bruker_data tabell